tasks/main.yml

```
@@ -0,0 +1,10 @@
+---
+- import_tasks: install.yml
+  tags: ['pretix-install']
```

```suggestion
  tags: pretix-install
```
or
```suggestion
  tags:
    - pretix-install
```

same thing with the other tags